java/maven.hints/src/org/netbeans/modules/maven/hints/pom/UseReleaseOptionHint.java

```diff
@@ -164,7 +172,7 @@ private List<ErrorDescription> createHintsForParent(String prefix, POMComponent
             return List.of();
         }
 
-        if (source == target && source >= 9) {
+        if (source == target && source >= 8) { // ok since 3.13.0+ maps release to source/target on 8
```

Comment:
If I understand this correctly, there is a corner case with plugin version >= 3.6.0 (COMPILER_PLUGIN_VERSION) and < 3.13.0 on JDK 8 this will break. To be frank: this is a pretty small nitpick.

Reply:
right, there are a few situations where it will judge wrongly, but I hope that they occur rarely and the more time passes even rarer.

The problem why this can't be solved perfectly at the moment is that if the app does not set a version, NB would see the version of the bundled maven dist. So this code here is careful to run the query only selectively.